src/openrct2/scripting/ScriptEngine.h

```diff
@@ -133,6 +137,9 @@ namespace OpenRCT2::Scripting
         };
 
         std::unordered_map<std::string, CustomActionInfo> _customActions;
+#    ifndef DISABLE_NETWORK
+        std::vector<std::shared_ptr<ScSocketBase>> _sockets;
```

Comment:
Given that the storage is shared_ptr it might be better to use std::list considering iterator invalidation matters for the Update loop and there is no penalty here.